### PR TITLE
wishList added

### PR DESCRIPTION
**Pull request by SumitSaundadkar**
click on wishList  and please PR.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
             <a class="anchor" href="#">Cart Management</a>
         </li>
         <li class="pages-list">
-            <a class="anchor" href="">Wishlist Management</a>
+            <a class="anchor" href="./screens/wistList/wishlist.html">Wishlist Management</a>
```
Comment (amarphule) [inline index.html:22]: On home page, nav links are not visible properly, please check the contrast ratio

**Review by amarphule — CHANGES_REQUESTED** [review]
Every think looks good to me, please check home page links

File: index.html (hunk 1 above)
Comment (SumitSaundadkar) [reply index.html:22]: done

File: screens/wistList/wishlist.css [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+    --offline-color:#d4dde08c;
+    --primary-font-colour: #fff;
+    --btn-primary-color:#007BFF;
+    --btn-secondary-color:#6C757D;
+    --box-shadow:rgba(19, 1, 1, 0.822);
+    --border-color:hsl(39, 77%, 83%);
+  
+    
+    
+  }
```
Comment (deekshithmd) [inline screens/wistList/wishlist.css:17]: try to give consistent line gap between classes

File: style.css [hunk 3]
```diff
@@ -37,6 +37,6 @@ a{
     display: flex;
     flex-direction: row;
     flex-wrap: wrap;
-    background-color: var(--secondary-color);
+    background-color: var(--btn-secondary-color);
     border-bottom-left-radius: 1rem;
```
Comment (deekshithmd) [inline style.css]: using px for border-radius will be good

**Review by deekshithmd — APPROVED** [review]

File: style.css (hunk 3 above)
Comment (SumitSaundadkar) [reply style.css]: done

File: screens/wistList/wishlist.css (hunk 2 above)
Comment (SumitSaundadkar) [reply screens/wistList/wishlist.css:17]: done

**Review by TusharJade — COMMENTED** [review]
looks good to me👍

**Pull request merged**